base/special/trig.jl

```diff
@@ -1254,7 +1254,11 @@ Simultaneously compute the sine and cosine of `x`, where `x` is in degrees.
 !!! compat "Julia 1.3"
     This function requires at least Julia 1.3.
 """
-function sincosd(x::Real)
+sincosd(x::Number) = (sind(x), cosd(x))
```

Comment:
```suggestion
sincosd(x) = (sind(x), cosd(x))
```

Reply:
Not sure why you marked this as resolved?  Since this works for e.g. matrices, it should have no type.

Reply:
I don't remember. I probably mark them resolved before someone pointed out it could work on matrices.